virttest/asset.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, cfg, tmpdir=data_dir.get_tmp_dir(), raise_errors=False):
             # Config file is just a string, convert it to a python file like
             # object using StringIO
             else:
-                self.cfg = StringIO.StringIO(cfg)
+                self.cfg = StringIO(cfg)
```

Comment:
On python2 , I try import StringIO and from io import StringIO ,but get different result as follow:
```
[root@chunfulaptop avocado-vt]# python
Python 2.7.5 (default, May  3 2017, 07:55:04) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-14)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from io import StringIO
>>> f = StringIO('hello')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: initial_value must be unicode or None, not str
>>>

[root@chunfulaptop avocado-vt]# python
Python 2.7.5 (default, May  3 2017, 07:55:04) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-14)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import StringIO
>>> f = StringIO.StringIO('hello')
>>> f.getvalue()
'hello'
>>> 

```